### ISSUE TITLE
Check if IANA references pointing to RFC 4210 must be updated to rfc4210bis 

### ISSUE THREAD

**Issue by HBrock** [lamps-wg/cmp-updates#53]
IANA wrote:
Should those references be replaced with references to this document (this is typical)? For IANA’s purposes, this could be covered by a line that says something like “All existing references to RFC 4210 at https://www.iana.org/assignments/smi-numbers/smi-numbers.xhtml and https://www.iana.org/assignments/core-parameters/core-parameters.xhtml should be replaced with references to this document,” or “All existing references [...] except for the following registrations:”

**Comment by HBrock** [comment]
SMI Security for PKIX Module Identifier
- id-mod-cmp: keep RFC 2510
- id-mod-cmp2000: keep RFC 4210
- BTW: should RFC4211 be the reference for id-mod-crmf2005?
- id-mod-cmp2000-02: keep RFC 5912
- id-mod-cmp2021-88: keep RFC 9480
- id-mod-cmp2021-02: keep RFC 9480

SMI Security for PKIX Extended Key Purpose
- id-kp-cmKGA: RFC 9480 -> RFC XXXX

SMI Security for PKIX CMP Information Types
- id-it-caProtEncCert: RFC 2510 -> RFC XXX
- id-it-signKeyPairTypes: RFC 2510 -> RFC XXX
- id-it-encKeyPairTypes: RFC 2510 -> RFC XXX
- id-it-preferredSymmAlg: RFC 2510 -> RFC XXX
- id-it-caKeyUpdateInfo: RFC 2510 -> RFC XXX
- id-it-currentCRL: RFC 2510 -> RFC XXX
- id-it-unsupportedOIDs: RFC 4210 -> RFC XXXX
- id-it-keyPairParamReq: RFC 4210 -> RFC XXXX
- id-it-keyPairParamRep: RFC 4210 -> RFC XXXX
- id-it-revPassphrase: RFC 4210 -> RFC XXXX
- id-it-implicitConfirm: RFC 4210 -> RFC XXXX
- id-it-confirmWaitTime: RFC 4210 -> RFC XXXX
- id-it-origPKIMessage: RFC 4210 -> RFC XXXX
- id-it-suppLangTags: RFC 4210 -> RFC XXXX
- id-it-caCerts: RFC 9480 -> RFC XXXX
- id-it-rootCaKeyUpdate: RFC 9480 -> RFC XXXX
- id-it-certReqTemplate: RFC 9480 -> RFC XXXX
- id-it-rootCaCert: RFC 9480 -> RFC XXXX
- id-it-certProfile: RFC 9480 -> RFC XXXX
- id-it-crlStatusList: RFC 9480 -> RFC XXXX
- id-it-crls: RFC 9480 -> RFC XXXX

SMI Security for PKIX CRMF Registration Controls
- id-regCtrl-altCertTemplate: RFC 4210 -> RFC XXXX
- id-regCtrl-algId: RFC 9480 -> RFC XXXX
- id-regCtrl-rsaKeyLen: RFC 9480 -> RFC XXXX

**Comment by HBrock** [comment]
@russhousley may I ask for your advice?
IANA asked me to specify which references shall be updated to the new RFC 4210bis and 6712bis. Above I provided a list. But I am uncertain if it is correct to update also references to the RFCs, where the OIDs were originally registered.
For example I found OIDs with reference RFC 2510, even though the OID is still in use in RFC 4210. I had expected that this reference should have been updated to RFC 4210 back then.
